client/src/NavBar.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars } from '@fortawesome/free-solid-svg-icons';
+import logo from './images/Logo.jpg';
+import { IconClickedNavBar } from './IconClickedNavBar';
+import { Link } from 'react-router-dom';
+
+export type PageType = 'movieApp' | 'register' | 'sign-in' | 'sign-out';
```

Comment:
Since this NavBar only shows up if you are not signed in, I'm not sure you should ever have a `sign-out` PageType?

Indeed, it looks like you don't have any links below to `sign-out`.